Source/ASTextNode2.mm

```diff
@@ -131,8 +137,6 @@ @implementation ASTextCacheValue
   return layout;
```

Comment:
I like our version of the comment on line 120

Reply:
I'm not sure I see a [different comment](https://github.com/TextureGroup/Texture/pull/1986/files#diff-f0bc2d9462b9b80ab173d5be5a5ab43193a1d3e008b8fb2beca8b8ade73f81d6R203)